components/org.wso2.carbon.identity.scim2.common/src/main/java/org/wso2/carbon/identity/scim2/common/impl/SCIMUserManager.java

```diff
@@ -5336,14 +5336,27 @@ public void updatePermissionListOfGroup(String groupName, String[] permissionToA
 
     /**
      * Returns all attributes belong to the extension schema dialect.
+     * unless its mapped local dialect claim value "support by default value is false".
      *
      * @param scimClaimToLocalClaimMap
      * @return Map of attribute.
      */
     private Map<String, Attribute> getAllScimSchemaAttributes(Map<ExternalClaim, LocalClaim> scimClaimToLocalClaimMap) {
+        Map<String, Attribute> filteredFlatAttributeMap = new HashMap<>();
```

Comment:
add new line after method signature

Reply:
Fixed in commit ea496939fce4e9e1d63603ea1589b076fc5804d5